### PR TITLE
Moved hideSoftKeyboard method from ConsentSignatureStepLayout into Fo…

### PR DESCRIPTION
**Pull request by suomi35**
…rmBody

The keyboard has to be hidden before entering the ConsentSignatureStep or it will remain visible once we get there. There are no keyboard-focussable views in the signature step so the keyboard will not be shown again once at that step.

### CODE REVIEW

File: backbone/src/main/java/org/researchstack/backbone/ui/step/body/FormBody.java [hunk 1]
```diff
@@ -96,4 +102,14 @@ private StepBody createStepBody(QuestionStep questionStep) {
             throw new RuntimeException(e);
         }
     }
+
+    private void hideSoftKeyboard() {
```
Comment (guillermoprados) [inline backbone/src/main/java/org/researchstack/backbone/ui/step/body/FormBody.java]: we could send the view group as parameter here, so we don't need to hold the reference to the parent in the class, at least we need it for something else, but It doesn't seems to

Comment (suomi35) [reply backbone/src/main/java/org/researchstack/backbone/ui/step/body/FormBody.java]: Nicely done. I missed you :)
I also added a null check.

Comment (guillermoprados) [reply backbone/src/main/java/org/researchstack/backbone/ui/step/body/FormBody.java]: haha. Well, I'm not familiarized with the ImputMethodManager, but this seems to be a method that could be added as a public static method in a utils class?

Comment (suomi35) [reply backbone/src/main/java/org/researchstack/backbone/ui/step/body/FormBody.java]: Ja! There was already a method in ViewUtils...though it doesn't work.
I added our in there too with a different signature. It works nicely.

**Pull request merged**